atlasdb-jepsen-tests/src/main/java/com/palantir/atlasdb/jepsen/events/EventVisitor.java

```diff
@@ -15,8 +15,12 @@
  */
 package com.palantir.atlasdb.jepsen.events;
 
+import java.util.List;
+
 public interface EventVisitor {
     void visit(InfoEvent event);
     void visit(InvokeEvent event);
     void visit(OkEvent event);
+    boolean valid();
+    List<Event> errors();
```

Comment:
These should probably be on a separate interface. It's a bit weird for `EventVisitor` to return `valid` and `errors()`